docs/contributorfile.rst

```diff
@@ -116,12 +126,26 @@ At physiopy, we use Issues and Milestones to keep track of and organise our work
 Labels
 ------
 The current list of labels are `here <https://github.com/physiopy/phys2bids/labels>`_. They can be used for **Issues**, **PRs**, or both.
+We use `auto <https://github.com/intuit/auto>`_ to automate our semantic versioning and Pypi upload, so **it's extremely important to use the right PR labels**!
+
+.. _issueprlabels:
+
+Issue & PR labels
+~~~~~~~~~~~~~~~~~~~
+-  Documentation: Improvements or additions to documentation. This category includes (but is not limited to) docs pages, docstrings, and code comments.
```

Comment:
This is a minor thing, but it would be awesome if the labels themselves could be replaced with badges.

Reply:
Here's an example (built [here](https://shields.io/category/issue-tracking)): https://img.shields.io/github/labels/physiopy/phys2bids/Documentation?style=plastic

Reply:
> This is a minor thing, but it would be awesome if the labels themselves could be replaced with badges.

You mean in the bullet list?

Reply:
Yeah, just in the documentation.

Reply:
And the badges could probably link to the list of issues with each label, if we want.

Reply:
That would be very convenient actually! I like that.

Reply:
Yeah I wondered how did you do it in `tedana`, cause I definitely wanted to do it here!
I wouldn't stall this PR for this reason, but we can definitely open an issue about it!

Reply:
(Also I would use the _flat_ style rather than the plastic one)